app/views/notifications/index.html.haml

```diff
@@ -0,0 +1,14 @@
+%h3 Your Notifications
+%hr
+%br
+
+.older-notifications
+  - @older_notifications.each do |notification|
+    = link_to project_path(notification.notifiable) do
+      %p
+        = image_tag notification.notifiable.avatar.url(:thumb), width: 40
+        #{ notification.actor.name } #{ notification.action }
```

Comment:
Please don't add space inside the insterpolation, this should be `#{notification.actor.name} #{notification.action}`